pycbc/events/significance.py

```diff
@@ -187,6 +187,19 @@ def get_n_louder(back_stat, fore_stat, dec_facs,
     Wrapper to find the correct n_louder calculation method using standard
     inputs
     """
+    # This nan to inf conversion is a safety shield; there shouldn't be any
+    # nan statistic values, but if there are, they should be considered as
+    # quiet as possible. nans are considered greater than any floats in an
+    # argsort
+    nanmask = np.isnan(back_stat)
+    if any(nanmask):
+        logging.warning(
+            "Removing %d NaN background statistic values",
+            np.count_nonzero(nanmask),
```

Comment:
If all you want to do is get the number of True's in a Boolean array `nanmask.sum()` is simplest and fastest